src/pyscipopt/scip.pyx

```diff
@@ -1706,6 +1776,18 @@ cdef class Model:
         """Print statistics."""
         PY_SCIP_CALL(SCIPprintStatistics(self._scip, NULL))
 
+    def writeStatistics(self, filename="origprob.stats"):
+      """Write statistics to a file.
+
+      Keyword arguments:
+      filename -- name of the output file
+      """
+      # use this doubled opening pattern to ensure that IOErrors are
+      #   triggered early and in Python not in C,Cython or SCIP.
+      with open(filename, "w") as f:
+          cfile = fdopen(f.fileno(), "w")
+          PY_SCIP_CALL(SCIPprintStatistics(self._scip, cfile))
+
```

Comment:
I would prefer to move this functionality into the existing method `printStatistics()` and use the filename as optional parameter. If it's not set, the statistics are printed to the console.

Reply:
hm, I would not recommend this as then the function name is inconsistent, although I can understand the desire to keep the interface lean.
print = output to screen
write = output to file

I have to say that I find the structure/naming conventions of SCIP itself rather convoluted. I personally would never have named a function print if it wrote to file.

Reply:
I get your point. It's less "SCIPy", though,... but this does not necessarily have to be a bad thing ;-) @fserra what do you think? I'd be fine with a new function `writeStatistics()`.

Reply:
I have never thought of the difference between write and print, but I have to say that I agree with @penenkel. I guess we can depart from the SCIP way of things here while keeping the interface consistent. However, in general, I wouldn't depart too much from SCIP.